setup.py

```diff
@@ -61,22 +61,22 @@
       ]
 
 setup(name='ibmdbpy',
-      version='0.1.4',
+      version='0.1.4.post3',
```

Comment:
Post-releases are not meant for bug fixes (see [PEP 440](https://www.python.org/dev/peps/pep-0440/#post-releases), we should just bump this to 0.1.5.

Reply:
Ok, I've made the change in doc/source/conf.py as well.